src/Multiselect.vue

```diff
@@ -288,89 +344,120 @@ export default {
      */
     showNoOptions: {
       type: Boolean,
-      default: true
+      default: true,
     },
     showNoResults: {
       type: Boolean,
-      default: true
+      default: true,
     },
     tabindex: {
       type: Number,
-      default: 0
-    }
+      default: 0,
+    },
+  },
+  watch: {
+    value: {
+      // the watch will look to any changes in value to adjust selectedOptions for adding and removing values
+      immediate: true,
+      handler(newVal, oldVal) {
+        if (newVal && oldVal) {
+          if (newVal.length > oldVal.length) {
+            this.selectedOptions.push(newVal[newVal.length - 1]);
+          } else {
+            this.selectedOptions = this.selectedOptions.filter((option) =>
+              newVal.includes(option)
+            );
+          }
+        }
```

Comment:
`value` is coming from outside, so it can also be changed to a completely different array, not just adding and removing values. To handle any possible change of `value` it should IMO look something like this:

```suggestion
      // the watch will look to any changes in value to adjust selectedOptions
      immediate: true,
      handler(newVal) {
        if (newVal && JSON.stringify(newVal) !== JSON.stringify(this.selectedOptions)) {
            this.selectedOptions = [...newVal];
        }
```